### PR TITLE
The Deep Scan Device

### PR DESCRIPTION
**Pull request by klaasjared**
Tired of the ore detector sprite that is so old that you could practically smell the dust coming off it? Well, fear not! Introducing the Deep Scan Device, the new and improved method for scanning ores deep underground!

This PR adds a new ore detector sprite and renames the device to better suit the functionality.

An image of the device is included below.

Screenshot 2021-02-28 211634

### CODE REVIEW

File: code/modules/mining/drilling/scanner.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /obj/item/weapon/mining_scanner
-	name = "ore detector"
+	name = "Deep Scan Device"
```
Comment (MistakeNot4892) [inline code/modules/mining/drilling/scanner.dm:2]: This isn't a proper noun, should be lowercase.

Comment (klaasjared) [reply code/modules/mining/drilling/scanner.dm:2]: Thank you - bit finicky with that stuff

**Review by MistakeNot4892 — COMMENTED** [review]
Looks good to me.

**Comment by TheFurryFeline** [comment]
Also an empty file here for the icon? What's going on there? Icondiff shows the addition but not when viewing the files changed part of the PR... 🤔

**Pull request merged**